src/main/java/com/courtcanva/ccfranchise/controllers/FranchiseeController.java

```diff
@@ -46,6 +41,12 @@ public SuburbListAndFilterModeGetDto addDutyAreas(@RequestBody @Valid SuburbList
         return franchiseeService.dutyAreas(suburbListAndFilterModePostDto, franchiseeId);
     }
 
+    @GetMapping("/{franchiseeId}/orders/accepted")
+    public OrderAcceptedAndCompletedPaginationGetDto acceptedOrdersList(@PathVariable(value = "franchiseeId") Long franchiseeId,
+                                                                        @RequestParam(value = "page") int pageNumber) {
+        return franchiseeService.findFranchiseeAcceptedOrders(franchiseeId, pageNumber);
```

Comment:
I suggest put the logic in `OrderService `instead of `FranchiseeService`, cause it's deals with order.s

Reply:
Find orders should be in the order service

Reply:
In the franchiseeService, I get the franchisee Entity firstly, then the orders are bound with franchisee. I use Franchisee controller to call franchisee service ,then use franchisee service to call order service.